src/platform/user/profile/vap-svc/actions/transactions.js

```diff
@@ -132,11 +132,18 @@ export function refreshTransaction(
             transactionRefreshed?.data?.attributes?.metadata?.[0] ?? {};
           const errorCode = errorMetadata.code ?? 'unknown-code';
           const errorKey = errorMetadata.key ?? 'unknown-key';
+          const addressSection =
```

Comment:
We may want to keep recording `address`, not `mailingAddress` and `residentialAddress` to keep our analytics consistent with what we were previously recording. I am not very opinionated on this and could also record both address section names instead.

Reply:
I'm good with it, I think it's a nice addition

Reply:
Let's check with @Samara-Strauss and @mshea0606 since they are the ones who look at Google Analytics the most.

To be clear, we're considering being more granular with the keys we send to Google Analytics. So instead of seeing entries like `ADDR301_AddressPOU-address-save-failure`, we'd see separate entries for `ADDR301_AddressPOU-mailingAddress-save-failure` and `ADDR301_AddressPOU-residentialAddress-save-failure`

![image](https://user-images.githubusercontent.com/20728956/102498282-9ce72780-402e-11eb-9558-a833adb93d16.png)

Reply:
I'm fine either way, but think we should be more granular since this change _is_ going to introduce recording granular events for phone number failures. Might as well be consistent. On the flip side, I think we are probably more concerned with the _types_ of errors we're seeing from VA Profile and recording more granular events will make that harder to reason about.